bootstrap.sh

```diff
@@ -97,6 +97,7 @@ STABLE_ROOT_PATH=$PWD/workspace
 NEXTCLOUD_AUTOINSTALL_APPS="${NEXTCLOUD_AUTOINSTALL_APPS[@]}"
 DOCKER_SUBNET=192.168.21.0/24
 PORTBASE=821
+XDEBUG_MODE=develop
```

Comment:
Maybe we can still stick to off by default as this might only be relevant to backend devs

Reply:
develop will just enhance the error messages to be a bit more readable not step debugging. But yes, I can change this also to `none` if you prefer.

Reply:
Ah right, sorry mixed that up.